pkg/features/kube_features_test.go

```diff
@@ -73,3 +73,21 @@ func TestAllRegisteredFeaturesExpected(t *testing.T) {
 		}
 	}
 }
+
+func TestEnsureAlphaGatesAreNotSwitchedOnByDefault(t *testing.T) {
+	registeredFeatures := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()
```

Comment:
I think we want to iterate over `defaultKubernetesFeatureGates` and `defaultVersionedKubernetesFeatureGates` rather than extracting just the current version of gates from the global

Reply:
`defaultKubernetesFeatureGates` seems empty at this point, but added it for safety's sake.